catalog/task/create-internal-request/0.6/README.md

```diff
@@ -0,0 +1,35 @@
+# create-internal-request
+
+Creates an InternalRequest resource to call IIB service
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| pipelineRunName | The name of the Parent PipelineRun for this task | No | `ir-$(context.pipelineRun.name)` |
+| request | Internal pipeline request name | No | |
+| params | Internal Request parameters | No | |
+| inputDataFile | Optional file to read data from | Yes | "" |
+| updateGenericResult | Should the task update the genericResult result  | Yes | "false" |
+| requestUpdateTimeout | Max seconds to wait until the status is updated | Yes | 360 |
+
+## Changelog
+
+### changes since 0.5
+- adds `genericResult` result to expose a generic result of this task to the Pipeline
```

Comment:
Can you say that it is the generic result from the internal request CR? `generic result` is an extremely broad english term, I think it would be improved to say at least where it comes from

Reply:
I rewrote the text only saying what it means for the task.